Subsystem/Elevator

CoderLiam926 opened the pull request:
(Elevator)

kjrokos commented on src/main/java/frc/robot/subsystems/ElevatorSubsystem.java:
```diff
@@ -1,9 +1,15 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ElevatorSubsystem extends SubsystemBase {
+SparkMax elevatorMotor = new SparkMax(13, MotorType.kBrushed);
+SparkMax dealgaeFlipper = new SparkMax(13, MotorType.kBrushed);
+SparkMax dealgaeWheels = new SparkMax(13, MotorType.kBrushed);
```
Make sure these are different IDs, and add to constants file

CoderLiam926 replied:
I did it! (At least I think it was me!)

Jalasta20XX commented on src/main/java/frc/robot/subsystems/ElevatorSubsystem.java:
```diff
@@ -1,9 +1,15 @@
 
 public class ElevatorSubsystem extends SubsystemBase {
+SparkMax elevatorMotor = new SparkMax(13, MotorType.kBrushed);
+SparkMax dealgaeFlipper = new SparkMax(13, MotorType.kBrushed);
+SparkMax dealgaeWheels = new SparkMax(13, MotorType.kBrushed);
```
```suggestion
SparkMax deAlgaeFlipper = new SparkMax(13, MotorType.kBrushed);
SparkMax deAlgaeWheels = new SparkMax(13, MotorType.kBrushed);
```

kjrokos replied:
I agree - I had the same thought. You could also go a different route with something like `AlgaeRemoverFlipper`

CoderLiam926 replied:
Me and Henry think the original name works the best.

CoderLiam926 replied:
No one has responded yet...

CoderLiam926 replied:
Okay, we changed it!

Jalasta20XX reviewed (approved):
I had a hard time parsing what this said at first, I'd prefer if it was more clearly capitalized.

Henyahyah reviewed (approved):
Looks fine! Eventually we'll want to add controller inputs once we figure out the details of what we want to happen and how we want it to happen.

CoderLiam926 committed:
Merge branch 'main' into Subsystem/Elevator

CoderLiam926 committed:
I properly filled the commands in the Elevator Subsystem

(Not tested or mentored but built successfully)

Henyahyah committed:
Updating SparkMax IDs and adding motor

Same fixes as what I did in the cage branch. The elevator requires two motors, so I added another.

/!\ I didn't add any code using the second motor yet. /!\

Henyahyah reviewed (approved):
cool

CoderLiam926 committed:
Merge branch 'main' into Subsystem/Elevator

CoderLiam926 committed:
Merge branch 'main' into Subsystem/Elevator
